setup.cfg

```diff
@@ -8,6 +8,8 @@ exclude=tests,migrations
 [mypy]
 ignore_missing_imports=true
 strict_optional=true
+disallow_any_generics=true
```

Comment:
What is an example of what this would disallow?

Reply:
`foo: List` rather than `foo: List[int]`